deployments/properties/azure-kubernetes-service-helm.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to AKS with Helm",
+    "description": "Deploy an application to an Azure Kubernetes Service Cluster using Azure Credentials and Helm",
+    "creator": "Microsoft Azure",
+    "iconName": "azure",
```

Comment:
Non blocking comment: Do you want to have an AKS specific icon. Possibly overlay with Helm as well.

Reply:
We're fine with the default Azure icon.